paasta_tools/setup_marathon_job.py

```diff
@@ -241,7 +241,8 @@ def log_bounce_action(line, level='debug'):
         at_risk_tasks=bounce_lib.flatten_tasks(old_app_at_risk_tasks),
     )
 
-    kill_given_tasks(client=client, task_ids=[task.id for task in tasks_to_kill], scale=True)
+    if tasks_to_kill:
+        kill_given_tasks(client=client, task_ids=[task.id for task in tasks_to_kill], scale=True)
```

Comment:
It seems better that `kill_given_tasks` knows how to do 'the right thing' and doesn't call `client.kill_given_tasks` if `tasks_to_kill` is empty.